src/components/sendAskMoney/SendAskMoney.vue

```diff
@@ -606,38 +632,37 @@
         this.searchName = ""
         this.activeClass = 0
         this.errors.transfer.balance = false
+        this.errors.transfer.amount = false
       },
 
       async newLinkTab() {
-        if (this.amountForCredit > 0) {
-          // This to ensure we are left with 2 decimals only
-          this.amountForCredit = this.amountForCredit.toFixed(2)
+        this.errors.credit.amount = false
+        if (this.amountForCredit <= 0) {
+          this.errors.credit.amount =
+            "Le montant à créditer doit être un nombre positif"
+          return
+        }
```

Comment:
It seems all these checks are ready to be done upon keystroke and not after clicking the `send`  button. Why not do this ?

Reply:
We might do this on another PR ?